.github/workflows/reusable-workflow.yml

```diff
@@ -0,0 +1,328 @@
+# Reusable workflow to run the following jobs:
+#
+#   - multiplatform-tests
+#     - [ ubuntu-22.04 | ubuntu-20.04 | windows-2019 | windows-2022 ]
+#     - [ Debug | Release ]
+#     - execute tests in different versions of ubuntu with different build types
+#
+#   - asan
+#     - ubuntu-22.04
+#     - execute tests with ASAN flag
+#
+#   - tsan
+#     - ubuntu-22.04
+#     - execute tests with TSAN flag
+#
+#   - clang
+#     - ubuntu-22.04
+#     - execute clang-tidy check
+#
+#   - coverage
+#     - ubuntu-22.04
+#     - execute test with coverage flag and upload results
+#
+#   - flaky
+#     - ubuntu-22.04
+#     - execute flaky tests
+#
+#   - docs
+#     - ubuntu-22.04
+#     - execute docs compile and tests
+#
+#   - uncrustify
+#     - ubuntu-22.04
+#     - test uncrustify
+#
+#   - python-linter
+#     - ubuntu-22.04
+#     - test python linter
+#
+
+name: reusable-workflow
+
+on:
+
+  workflow_call:
+    inputs:
+
+      custom_version_build:
+        description: >
+          Version of Fast DDS build from eProsima-CI.
+        required: true
+        type: string
+
+      dependencies_artifact_postfix:
+        description: >
+          Postfix name to add to artifact name to download dependencies.
+          This is use to download a specific artifact version from eProsima-CI.
+        required: true
+        default: '_nightly'
+        type: string
+
+env:
+  code_packages_names: 'ddsrouter_core ddsrouter_yaml ddsrouter_tool'
+  docs_packages_names: 'ddsrouter_docs'
+
+jobs:
+
+#####################################################################
+# TEST
+
+  multiplatform-tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        cmake_build_type:
+          - Release
+          - Debug
+        os:
+          - ubuntu-20.04
+          - ubuntu-22.04
+          - windows-2019
+          - windows-2022
+
+    steps:
+
+      - name: Sync repository
+        uses: eProsima/eProsima-CI/external/checkout@v0
+        with:
+          path: ${{ github.workspace }}/src
```

Comment:
```suggestion
          path: src
```

Reply:
Addressed in a074d49 (applied everywhere)